docs-website/src/pages/docs/sources-summary.jsx

```diff
@@ -0,0 +1,728 @@
+import React from "react";
+import Layout from "@theme/Layout";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import FilterBar from "./_components/FilterBar";
+import FilterCards from "./_components/FilterCards";
+import useGlobalData from "@docusaurus/useGlobalData";
+import DropDownFilter from "./_components/DropDownFilter";
+
+const metadata = [
```

Comment:
can we put this in JSON and then `import` it?

Reply:
yeah, will put into a son file